src/jira/util/jwt.ts

```diff
@@ -184,6 +184,19 @@ export const verifySymmetricJwtTokenMiddleware = (secret: string, tokenType: Tok
 
 const ALLOWED_BASE_URLS = [BASE_URL]
 
+const isStagingTenant = (req: Request): boolean => {
+	try {
+		const hostBaseUrl = req.body?.baseUrl;
+		if (hostBaseUrl) {
```

Comment:
You don't need the if at all.  It either parses or it throws an error.

Reply:
No, I am not going to use exceptions where condition can be used